frontend/src/pages/stakingOverview/Validator.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { TableRow, TableCell, Avatar, Box, Typography } from '@mui/material';
+
+export const Validator = (props) => {
```

Comment:
You should add propTypes [ref](https://legacy.reactjs.org/docs/typechecking-with-proptypes.html#proptypes).

Reply:
done